packages/openapi-ts/src/utils/write/schemas.ts

```diff
@@ -295,21 +266,8 @@ const modelToJsonSchema = (config: Config, model: Model) => {
 
 const exportSchema = (config: Config, model: Model) => {
     const jsonSchema = modelToJsonSchema(config, model);
-    const statement = ts.factory.createVariableStatement(
-        [ts.factory.createModifier(ts.SyntaxKind.ExportKeyword)],
-        ts.factory.createVariableDeclarationList(
-            [
-                ts.factory.createVariableDeclaration(
-                    ts.factory.createIdentifier(`$${model.name}`),
-                    undefined,
-                    undefined,
-                    ts.factory.createAsExpression(jsonSchema, ts.factory.createTypeReferenceNode('const'))
-                ),
-            ],
-            ts.NodeFlags.Const
-        )
-    );
-    return tsNodeToString(statement);
+    const obj = compiler.types.object(jsonSchema);
```

Comment:
Here we are passing the Record<string, unknown> (or any other object) and the object compiler fucntion handles the rest.